eva-accession-core/src/test/java/uk/ac/ebi/eva/accession/core/io/AccessionedVcfFactoryTest.java

```diff
@@ -0,0 +1,254 @@
+package uk.ac.ebi.eva.accession.core.io;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import uk.ac.ebi.eva.commons.core.models.factories.CoordinatesVcfFactory;
+import uk.ac.ebi.eva.commons.core.models.factories.VariantVcfFactory;
+import uk.ac.ebi.eva.commons.core.models.pipeline.Variant;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;
+
```

Comment:
Could you let us know what were the newly added tests in this file compared to the one from variation-commons?

Reply:
The only new test I have added is **"testReadContextBaseVariant"**. 
Rest All I kept as it is from CoordinatesVcfFactory just to make sure the code works as before in other cases.